### ISSUE TITLE
4.0.3 No Chromecast icon in Windows 10 (64-bit)

### ISSUE THREAD

**Issue by RobFreundlich** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1942]
* **I have checked that there are no issues with similar or the same content**
    * There's one open issue, but it's for an older version and a different OS
    * There's a closed issue, but it's also for an older version, and it's about the icon only showing up when the window is maximized.  Mine never shows up
* **I have checked the FAQ (https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/wiki/FAQ) and the answer I am looking for is not there
    * Yup

* **I have double checked and can reproduce the issue**
    * Yup

**OS:** Windows 10 64-bit

**Issue Descriptions:**

I have no Chromecast icon, in the normal player or the mini-player.  This is the case whether my window is normal size or maximized.

**Steps to Reproduce:**

1. Run GPMDP
2. Observe that there's no Chromecast icon

**Notes:**

I saw in the other issues that I need to have the bonjour service installed and port 5353 not blocked by my firewall.  As far as I can tell, both conditions are met (see screenshots).

**debug_info.zip:** [debug_info.zip](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/602423/debug_info.zip)

**Screenshot of normal window:**
![nochromeicon-normal](https://cloud.githubusercontent.com/assets/7975013/20467251/fdd9135e-af51-11e6-9262-6501c5a26e55.png)

**Screenshot of maximized window:**
![nochromeicon-maximized](https://cloud.githubusercontent.com/assets/7975013/20467252/01acbb66-af52-11e6-85d7-be8b280f5311.png)

**Bonjour service:**
![image](https://cloud.githubusercontent.com/assets/7975013/20467273/6e33ab3c-af52-11e6-8f2d-526355f41adc.png)

**Firewall:**
![image](https://cloud.githubusercontent.com/assets/7975013/20467290/b8aba3b8-af52-11e6-8a75-8bc4afdcaa86.png)
![image](https://cloud.githubusercontent.com/assets/7975013/20467300/eaa56930-af52-11e6-9ccf-b556939d3d6f.png)

**Comment by MarshallOfSound** [comment]
@RobFreundlich Thanks for the detailed issue report 👍 

Can you try restarting the Bonjour service and restarting GPMDP.  If that still doesn't work can you follow the instructions in this comment 👍 

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/1292#issuecomment-231511635

**Comment by RobFreundlich** [comment]
mdns-test.exe showed a blank white window, so I went on and installed bonjour from the link you provided.  I had to go through a couple of attempts because the existing bonjour service was very tenacious - I kept getting told I didn't have privs to get rid of it, even though I'm an admin.  Ultimately, I had to uninstall it, reboot, install the one you provided, get told it didn't work, reboot again, and then bam - GPMDP had its Chromecast icon!

Oddly, even though the new bonjour was installed, it didn't show in the Services list.  So I rebooted once more, and now everything is great.  Bonjour is now a service again, and I have my Chromecast icon in GPMDP, **and** it casts properly to my kitchen speakers, which was the whole purpose of this in the first place.

Thanks for the quick response time!
